docs/proxies.md

```diff
@@ -51,6 +51,28 @@ proxy = httpcore.HTTPProxy(
 )
 ```
 
+## Proxy Connection Mode
+
+There are two types of HTTP proxy:
+
+1. Forwarding (HTTP [absolute-url](https://tools.ietf.org/html/rfc7230#section-5.3.2))
+2. Tunneling (HTTP [`CONNECT` method](https://tools.ietf.org/html/rfc7231#section-4.3.6))
+
+By default we forwarding plain http requests and tunneling https requests.
+You can change this behavior with `proxy_mode: httpcore.ProxyMode` parameter:
+```py
+import httpcore
+import base64
+
+proxy = httpcore.HTTPProxy(
+    proxy_url="http://127.0.0.1:8080/",
+    proxy_mode=httpcore.ProxyMode.TUNNEL,  # Forces HTTP requests also use Tunneling
```

Comment:
I guess let's just drop the comment here, since the docs are already giving us context...

```suggestion
    proxy_mode=httpcore.ProxyMode.TUNNEL,
```